### ISSUE TITLE
Add Windows installation instructions

### ISSUE THREAD

**Issue by lucperkins** [buildpacks/docs#7]
Installation instructions have recently been added for macOS and Linux but Windows instructions are still missing and consist only of a URL for a ZIP file.

**Comment by zmackie** [comment]
@micahyoung You have a windows VM lying around you could use to update this?

**Comment by dfreilich** [comment]
@lucperkins Is this still relevant? What additional instructions would you like to see here? https://buildpacks.io/docs/install-pack/#windows

**Comment by lucperkins** [comment]
@dfreilich It'd be nice to have the Windows instructions directly analogous to the macOS/Linux instructions, i.e. an explicit set of commands required to make the executable available.

**Comment by jromero** [comment]
We are investigating https://chocolatey.org/ as the package manager of choice for distribution of the `pack` binary on Windows. How does that influence this issue?

**Comment by lucperkins** [comment]
Chocolatey installation would be fine, I think. I just think it'd be nice to have parity in the installation instructions, that's all. I'm not a Windows user myself but I do know that Windows folk often feel like an afterthought in the "cloud native" universe.

**Comment by dfreilich** [comment]
@jromero I wonder whether we should look into winget, now that Windows is pursuing package management (https://docs.microsoft.com/en-us/windows/package-manager/)

**Comment by dfreilich** [comment]
An update - we have pushed `v0.12.0` to chocolatey. It's currently waiting on chocolatey to review it (https://chocolatey.org/packages/pack), but once it's added to the site, we'll add it to the instructions.
